dll/win32/shell32/wine/shell32_main.h

```diff
@@ -44,6 +44,9 @@ BOOL SIC_Initialize(void);
 void SIC_Destroy(void) DECLSPEC_HIDDEN;
 BOOL PidlToSicIndex (IShellFolder * sh, LPCITEMIDLIST pidl, BOOL bBigIcon, UINT uFlags, int * pIndex) DECLSPEC_HIDDEN;
 INT SIC_GetIconIndex (LPCWSTR sSourceFile, INT dwSourceIndex, DWORD dwFlags ) DECLSPEC_HIDDEN;
+extern INT ShellLargeIconSize;
+extern INT ShellSmallIconSize;
```

Comment:
Trivial nit: put the variables in same order (small, large, bpp) as they appeared in the cpp file.